BitwardenShared/UI/Platform/Application/Support/Images.xcassets/Images/send_illustration.imageset/Contents.json

```diff
@@ -0,0 +1,25 @@
+{
+  "images" : [
+    {
+      "filename" : "send_illustration_light.pdf",
```

Comment:
@matt-livefront, i ended up going with `send_illustration` because we have an icon that is called `send` already.  Let me know if you have a better idea here though and im happy to make a change.

Reply:
Ah, one other option we could consider is to make an `Illustrations` folder and have it provide a namespace. Then we wouldn't have to worry about conflicts with icons and all illustrations would be grouped (`Asset.Images.Illustrations.send`).

<img width="263" alt="Screenshot 2024-10-09 at 9 36 20 AM" src="https://github.com/user-attachments/assets/47a9c22d-cdb1-450b-b22d-333e11531f3b">

Reply:
yeah i like that!

Reply:
if its cool with you, i can make this change in this [tickets](https://bitwarden.atlassian.net/browse/PM-12662) PR since I can see that being related to that work.

Reply:
Sounds good!